src/database/functions/GuildSettingsFunctions.ts

```diff
@@ -35,7 +35,7 @@ function GuildScheme(guildID: string) {
             welcomeChannelMessage: "Welcome {member} to {server}!",
             dmMessage: "Welcome {member} to {server}!",
             welcomeRoles: [],
-            CustomWelcomeBackground: ""
```

Comment:
if yo uchange it here you also need to change it in schemas GuildSettingsSchema.ts and interfaces global types